features/step_definitions/signup_definitions_step.rb

```diff
@@ -0,0 +1,19 @@
+Given("I am on the landing page") do
+  visit root_path 
+end
+
+When("I click on the {string} button") do |string|
+  click_on "Sign up"
+end
+
+Then("I fill in {string} with {string}") do |name, user_name|
+  fill_in(name, with: "Bob")
```

Comment:
Change the variable names, the first one is better described as `field` and the second one `content` then write the step like this.
``` 
fill_in field, with: content
```